WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModelTests.swift

```diff
@@ -249,6 +249,80 @@ final class StoreOnboardingViewModelTests: XCTestCase {
                                                               .init(isComplete: false, type: .customizeDomains)])
     }
 
+    func test_launch_store_task_is_marked_as_complete_for_already_public_store() async throws {
+        // Given
+        sessionManager.defaultSite = .fake().copy(isWordPressComStore: true, isBlogPublic: true)
+        sessionManager.defaultRoles = [.administrator]
```

Comment:
just wondering, is the admin role a requirement as well?

Reply:
Removed in 37272dfa058522769e3774e0521710b30e7b5925